client/components/data/query-reader-feeds-search/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFeedSearch } from 'state/reader/feed-search/actions';
+
+class QueryFeedSearch extends Component {
+	static propTypes = {
+		query: PropTypes.string,
+		searchFeeds: PropTypes.func,
+	}
+
+	componentWillMount() {
```

Comment:
ah, its because I fixed this issue in the followup connecting PR. https://github.com/Automattic/wp-calypso/pull/12560/files#diff-84fe6e8ba43f2d92ac0fe35f414dc846R12